### PR TITLE
Add mark-walle-victoria.md file

### LINKED ISSUE #19: Create a new file for mark-walle Victoria, BC
- [x] Create a branch
- [x] Add Commits
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by mark-walle**
closes #19

### CODE REVIEW

File: mark-walle-victoria.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Victoria, BC
+
+### Food Recommendations
+- Big Wheel Burger
+- Tacofino
+- Standard Pizza
+
+### Places to Go
+- Gonzales Hill Observatory
+- Ogden Point Breakwater
```
Comment (hectorsector) [inline mark-walle-victoria.md:10]: I should know what a Breakwater is... but I don't. Care to clarify? @mark-walle

Comment (mark-walle) [inline mark-walle-victoria.md]: It's a a man made spit made to protect the Victoria harbor from any strong currents or waves that might pass by.

Comment (mark-walle) [reply mark-walle-victoria.md:10]: It's a a man made spit made to protect the Victoria harbor from any strong currents or waves that might pass by.

**Review by hectorsector — APPROVED** [review]
Looks great 👍

**Pull request merged**